caniusepython3/test/test_dependencies.py

```diff
@@ -18,11 +18,19 @@
 from caniusepython3 import dependencies
 from caniusepython3.test import mock, unittest
 
-import io
-
 
 class GraphResolutionTests(unittest.TestCase):
 
+    def test_circular_dependencies(self):
+        reasons = {
+            'A': 'C',
+            'B': 'C',
+            'C': 'A',
```

Comment:
fwiw, I was attempting to replicate the exact cause that I saw IRL, substituting real package names for the names pre-existant in the tests.